src/screens/DashboardTx.js

```diff
@@ -26,11 +27,11 @@ class DashboardTx extends React.Component {
 
   componentDidMount = () => {
     this.getInitialData();
-    hathorLib.WebSocketHandler.on('network', this.handleWebsocket);
```

Comment:
question: This change seems reasonable to me, but it wasn't informed on the acceptance criteria and it doesn't seem directly related to the lib upgrade.

Is this related to the context of this PR?

Reply:
It is part of the same upgrade because we no longer have `WebSocketHandler` in the lib. However, we do have this class in the explorer itself, so I just changed it to use the explorer one.